packages/@aws-cdk/aws-lambda/README.md

```diff
@@ -254,6 +255,47 @@ number of times and with different properties. Using `SingletonFunction` here wi
 For example, the `LogRetention` construct requires only one single lambda function for all different log groups whose
 retention it seeks to manage.
 
+### Using Docker with Asset Code
```

Comment:
This is not about docker... This is about code bundling. Docker is just the mechanism we use to allow you to specify your bundling instructions.